modules/chttrans/chttrans.cpp

```diff
@@ -213,6 +213,23 @@ void Chttrans::save() {
     safeSaveAsIni(config_, "conf/chttrans.conf");
 }
 
+const Configuration *Chttrans::getConfig() const {
+#ifdef ENABLE_OPENCC
+    std::vector<std::string> profiles;
+    StandardPath::global().scanFiles(
```

Comment:
standardPath.multiOpen and filter::Suffix

Reply:
With multiOpen you will also get the file name sorted for free. And if you want, you can try to extract the name from json. For parsing JSON a few different options to think about:
1. Boost, used by libime already, probably not being built for android/mac, but should be easy by just tuning some flag.
2. json-c, used by Fcitx 5 already, but only for keyboard (Disabled on android/mac), so you'll need to build it on non-linux platform.

Reply:
Talking about Android, what's your opinion? @rocka

Reply:
It seems that Boost.JSON supports header-only mode: https://www.boost.org/doc/libs/1_85_0/libs/json/doc/html/json/overview.html#json.overview.requirements.header_only
so I guess it could be an easier option...